src/coreclr/src/libraries-native/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CORECLR_BUILD ON)
+
+if(CLR_CMAKE_HOST_UNIX AND NOT DEFINED CLR_CROSS_COMPONENTS_BUILD)
```

Comment:
Please remove the DEFINED here. It is not necessary. We define the variable to 1 (when the cross components build is on) and when it is not defined, it behaves as 0 or False in a condition.
I can see that we use both forms in the cmake files and I would like to standardize on the simpler form.

Reply:
Sounds good. I'll remove it, thanks.